docs-metadata/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "docs-metadata",
 	"displayName": "docs-metadata",
 	"description": "Simplifies editing metadata within docs.",
-	"version": "0.0.125",
+	"version": "1.0.0",
 	"publisher": "docsmsft",
 	"icon": "images/docs-logo-ms.png",
 	"preview": true,
```

Comment:
@adunndevster , should this still be preview?

Reply:
Oh it should not be!